src/offliner.js

```diff
@@ -228,6 +228,9 @@
       case 'xpromise':
         this._receiveCrossPromise(msg.id, msg.order);
         break;
+      case 'checkForActivationPending':
```

Comment:
Although now that I read through the rest of the code and see other uses, I'd stick with "checkForActivationPending", as it reuses the order of the words in symbols like "isActivationPending" and "activation-pending".